### ISSUE TITLE
Behavioural arrest seizure

### ISSUE THREAD

**Issue by oliviajhenry** [obophenotype/human-phenotype-ontology#5837]
**Preferred term label:**
Behavioural arrest seizure

**Synonyms**
Behavioral arrest seizure, hypokinetic seizure, unclassified behavioural arrest seizure, unclassified behavioral arrest seizure, unclassified hypokinetic seizure

**Definition (free text, please give PubMed ID)**
28276060

**Parent term (use hpo.jax.org/app)**
Nonmotor seizure (new term request, child of seizure: HP:0001250)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Unknown onset epilepsy

**Your nano-attribution (ORCID)**

**Comment by oliviajhenry** [comment]
Additional definition information:
A type of unknown onset non-motor seizure characterised by an arrest or pause of activities, freezing, or immobilisation as the predominant semiological feature throughout the seizure.
PMID: 28276060

**Comment by DLewisSmith** [comment]
Please see https://github.com/obophenotype/human-phenotype-ontology/issues/5838 as this good suggestion is incorporated within the same discussion of concepts of seizures irrespective of electroclinical onset and that do not have motor semiologies.

**Comment by pnrobinson** [comment]
This issue has been superceded by #5838
